ios/RNRadar.m

```diff
@@ -411,14 +411,17 @@ - (void)didLogMessage:(NSString *)message {
 RCT_EXPORT_METHOD(startTrip:(NSDictionary *)optionsDict resolve:(RCTPromiseResolveBlock)resolve reject:(RCTPromiseRejectBlock)reject) {
     // { tripOptions, trackingOptions } is the new req format.
     // fallback to reading trip options from the top level options.
-    NSDictionary *tripOptionsDict = optionsDict[@"tripOptions"];
+    NSMutableDictionary *tripOptionsDict = optionsDict[@"tripOptions"];
     if (tripOptionsDict == nil) {
         tripOptionsDict = optionsDict;
     }
-    RadarTripOptions *options = [RadarTripOptions tripOptionsFromDictionary:tripOptionsDict];
-    if (options.scheduledArrivalAt) {
-        options.scheduledArrivalAt = [RCTConvert NSDate:options.scheduledArrivalAt];
+    NSObject *scheduledArrivalAtObj = tripOptionsDict[@"scheduledArrivalAt"];
+    if (scheduledArrivalAtObj && [scheduledArrivalAtObj isKindOfClass:[NSNumber class]]) {
+        [tripOptionsDict setObject:[RCTConvert NSDate:scheduledArrivalAtObj] forKey:@"scheduledArrivalAt"];
```

Comment:
I think your original instinct was good - we do centralize these conversions in the ios and android sdk's so that they're consistent across all platforms.  [Android already supports passing in `long's`](https://github.com/radarlabs/radar-sdk-android/blob/master/sdk/src/main/java/io/radar/sdk/RadarTripOptions.kt#L70), we should probably do the same thing on iOS to keep them consistent.

Reply:
Ok, did that: https://github.com/radarlabs/radar-sdk-ios/pull/264